src/lib/cookie-utils.js

```diff
@@ -21,3 +22,12 @@ export const setCookie = (cookieName : string, cookieValue : string, expirationM
     const expires = `expires=${ d.toUTCString() }`;
     document.cookie = `${ cookieName }=${ cookieValue }; Path=/; ${ expires }`;
 };
+
+export const getUserIdCookie = () : ?string => {
```

Comment:
These methods were removed from 'tracker-component'. They have not been altered. (note: ONE_MONTH_IN_MILLISECONDS should probably be in constants)